eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -616,6 +616,14 @@
         <resource-config-dmr name="Bound Address"
                              path="/socket-binding-group=standard-sockets/socket-binding=http"
                              attribute="bound-address" />
+        <resource-config-dmr name="Immutable"
+                             path="/subsystem=hawkular-wildfly-agent"
```

Comment:
@jmazzitelli What will happen when we are monitoring a remote server (via remote-dmr) ?
Those won't always have hawkular-wildfly-agent.